playground/frontend/lib/config.g.dart

```diff
@@ -17,6 +17,8 @@
  */
 
 const String kAnalyticsUA = 'UA-73650088-2';
+const int kMatomoSiteId = 19;
+const String kMatomoEndpointUrl = 'https://analytics.apache.org/matomo.php';
```

Comment:
This also has to be added to `gradle.properties`, but these two files are out of sync already, so this is better be done in
- #21925